ui/src/main/webapp/src/app/components/form.component.ts

```diff
@@ -24,15 +24,17 @@ export class FormComponent {
     handleError(error:any) {
         this.errorMessages = [];
         if (!error) {
-            this.errorMessages.push("Server call failed!");
-        } else if (error.parameterViolations) {
+            this.errorMessages.push("Server call failed.");
+        }
+        else if (error.parameterViolations) {
             error.parameterViolations.forEach(violation => {
                 console.log("Violation: " + JSON.stringify(violation));
                 this.errorMessages.push(violation.message);
             });
-        } else {
+        }
+        else {
```

Comment:
As you say, there are no consistent best practices, so we may set our own. We already have coding style, it's described [here](https://github.com/windup/windup/blob/master/ide-config/Eclipse_Code_Format_Profile.xml).